common/src/main/java/org/keycloak/common/util/KeycloakUriBuilder.java

```diff
@@ -253,8 +253,8 @@ public KeycloakUriBuilder schemeSpecificPart(String ssp) throws IllegalArgumentE
 
         StringBuilder sb = new StringBuilder();
         if (scheme != null) sb.append(scheme).append(':');
-        if (ssp != null)
-            sb.append(ssp);
+//        if (ssp != null) //null case will cause exception thrown on above code
+        sb.append(ssp);
```

Comment:
Missing a description on why this is an issue, and if it is an issue we should ideal have a test-case for it, and remove the line rather than comment it out.